src/controller/ExamController.ts

```diff
@@ -71,10 +71,9 @@ export class ExamController {
     // find an exam by the given id
     async findById(request: Request, response: Response, next?: NextFunction) {
         try {
-            const exam = await this.examRepository.findOne({
+            return await this.examRepository.findOne({
```

Comment:
This is just shortening the code a little by getting rid of the unused variable

Reply:
ok Ill update the other PRs to have this change